cmd/protoc-gen-go-grpc/grpc.go

```diff
@@ -121,6 +167,7 @@ func genService(gen *protogen.Plugin, file *protogen.File, g *protogen.Generated
 		g.P(deprecationComment)
 	}
 	g.P("func New", clientName, " (cc ", grpcPackage.Ident("ClientConnInterface"), ") ", clientName, " {")
+	helper.preprocessServiceName(g, service, clientName)
```

Comment:
How would this be used?  Maybe name it `generateNewClientDefinitions` or something?

Reply:
Done